zb-db/src/main/java/io/zeebe/db/impl/rocksdb/RocksDbConfiguration.java

```diff
@@ -13,10 +13,18 @@
 
   public static final long DEFAULT_MEMORY_LIMIT = 512 * 1024 * 1024L;
   public static final int DEFAULT_UNLIMITED_MAX_OPEN_FILES = -1;
+  public static final int DEFAULT_MAX_WRITE_BUFFER_NUMBER = 6;
+  public static final int DEFAULT_MIN_WRITE_BUFFER_NUMBER_TO_MERGE = 3;
```

Comment:
do we need checks for these values?

Reply:
You mean if they are > 0, or in relation to each other?

Reply:
yeah the relation like what if min is larger then max or something

Reply:
I looked into it, and RocksDB will make sure the values aren't wrong - if you put a negative value, for example, it will set it to 1 for minWriteBufferToMerge. If you set `minWriteBufferToMerge > maxWriteBufferNumber` it will set `minWriteBufferToMerge` to 1. So it seems if it's invalid it sets it to the default, e.g. `maxWriteBufferNumber = -1` and RocksDB will use 2.

It sucks that it silently fails, but as these are experimental, I don't know how much validation we want to run on them (considering they will still work, just not maybe as intended). wdyt?

Reply:
For now as these are experimental config, I will leave it as is, as users should be very careful when modifying these settings.